CHANGELOG.md

```diff
@@ -194,6 +194,15 @@ Enabled nightly run of build and test via Github actions
 
 [dde3172](https://github.com/neomatrix369/nlp_profiler/commit/dde31723b7cb1c1105b9df828bf7429094113de4) [@neomatrix369](https://github.com/neomatrix369) _Sun Nov 14 09:12:33 2021 +0000_
 
+---
+### GitHub branch `grammar_check` Grammar_quality_check: language tool replaced with Gingerit
+
+Implemented functionality via PR [#69](https://github.com/neomatrix369/nlp_profiler/pull/69) - details described in the body of the PR.
+
+Replaced language tool with Gingerit for faster calculations
+
+[b5a5dda](https://github.com/neomatrix369/nlp_profiler/pull/69/commits/b5a5ddaad01f07230cf232712671d43dd9db9862) [@bitanb1999](https://github.com/bitanb1999) _Sun Nov 14 09:12:33 2021 +0000_
```

Comment:
Small fix: put today's date, you should see it in your commit info when you do `git show [commit hash]`